assets/training/forecasting_demand/automl-hts-train/components/automl_hts_train/spec.yaml

```diff
@@ -4,7 +4,7 @@ type: pipeline
 name: automl_hts_training
 display_name: AutoML Hierarchical Timeseries Forecasting - Training
 description: Enables AutoML Training for hts components.
-version: 0.0.2
+version: 0.0.3
```

Comment:
Sub-component version update required in this one.

Reply:
I have updated it. Thanks for catching this!